lib/laa_crime_schemas/structs/document.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module LaaCrimeSchemas
+  module Structs
+    class Document < Base
+      attribute :s3_object_key, Types::String
+      attribute :filename, Types::String
+      attribute :content_type, Types::String
+      attribute :file_size, Types::Coercible::Integer
```

Comment:
Does `file_size` need it's own type? I noticed the application reference which is also an integer [has a type definition](https://github.com/ministryofjustice/laa-criminal-legal-aid-schemas/blob/main/lib/laa_crime_schemas/types/types.rb#L8). Not sure what the motivation was around that

Reply:
I think it's fine so long as we're only dealing with whole numbers.